src/manager/agents/common-module/module/utils/common_utils.cpp

```diff
@@ -16,6 +16,51 @@
 namespace common_utils
 {
 std::map<std::string, ScAddr> CommonUtils::managerParametersWithAgentRelations;
+std::vector<std::vector<ScAddr>> CommonUtils::componentsClasses;
+
+ScAddr CommonUtils::CheckIfMyselfDecompositionAddrExists(ScMemoryContext & m_memoryCtx)
```

Comment:
```suggestion
ScAddr CommonUtils::GetMyselfDecompositionAddr(ScMemoryContext & m_memoryCtx)
```
В данном случае метод возвращает узел декомпозиции, а не проверяет. Методы, которые что-то проверяют, должны возвращать bool